src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -741,11 +741,17 @@ private VariantContext createVariantContextFromSampleRecord(final ExtractCohortR
         }
 
         final String callGT = sampleRecord.getCallGT();
-        if ("./.".equals(callGT)) {
-            genotypeBuilder.alleles(Arrays.asList(Allele.NO_CALL, Allele.NO_CALL));
+
+        String[] splitGT = callGT.split("[/|]");
+        // This should match against anything like ".", "./.", ".|.", "././.", ".|.|.", etc
+        if (callGT.matches("\\.([/|]\\.)*")) {
```

Comment:
would this not suffice or is there something weird you're specifically trying to keep out?

```suggestion
        if (callGT.contains(".")) {
```

Reply:
might be worth noting that it matches no-calls, but not hemis